### DISCUSSION TITLE
Tips about training dreamboth for specific actions

### DISCUSSION THREAD

**Discussion by ClaraSanders** [TheLastBen/fast-stable-diffusion#2193] [Q&A]
Now that Google is no longer causing issues (fingers crossed), I've always wondered how one can train an AI like SD to perform specific actions. It might seem obvious, but it's not clear to me. For instance, when training it to recognize faces, you typically need to describe the details of the captioned pictures in a general manner, such as 'a woman with a white shirt smiling on the street.' However, when training it for a specific art style, you need to be more precise, like 'a mature woman with long black hair wearing a white shirt, smiling on the street.' But what if I want to train SD to recognize people properly smoking a cigarette? Should I explicitly mention the act of smoking, or should I describe everything except the cigarette and the hand holding it? Similarly, what if I want to train it to recognize people playing the violin or riding a bicycle?

**Comment by TheLastBen** [comment]
you really don't need to captions your images, the model is smart enough to segment the images and classify them without the help of captions, so I recommend to stick to one single token that will be used to summon the style/concept in the prompt.
